lib/auth0Bundler.js

```diff
@@ -11,8 +11,19 @@ const buildLiteralAst = require('./buildLiteralAst');
 
 const dependencies = { rollup, babelTransform, BabelPluginExportToFunction, buildLiteralAst, rollupCommonjs };
 
-module.exports = {
-    bundleScript: R.partial(bundle, [ dependencies, { commonjs: false } ]),
-    bundleRule: R.partial(bundle, [ dependencies, { commonjs: false } ]),
-    bundleHook: R.partial(bundle, [ dependencies, { commonjs: true } ])
+const defaultNodeVersion = 4;
+
+module.exports = function createBundler({ nodeVersion = defaultNodeVersion } = {}) {
+    const parseNodeVersion = parseInt(nodeVersion, 10);
```

Comment:
Can we validate the nodeVersion against an array of valid node versions? This way we could ensure that it is a number and only let node versions that are supported by Auth0 pass.